odoo/addons/base/tests/test_ir_actions.py

```diff
@@ -62,7 +62,7 @@ class TestServerActions(TestServerActionsBase):
 
     def test_00_action(self):
         self.action.with_context(self.context).run()
-        self.assertEqual(self.test_partner.comment, 'MyComment', 'ir_actions_server: invalid condition check')
+        self.assertEqual(self.test_partner.comment, '<p>MyComment</p>', 'ir_actions_server: invalid condition check')
```

Comment:
It is actually strange that writing MyComment inserts ``<p>MyComment</p>``. I suppose this is due to html fields always containing valid html. I would suggest to updater server action to match standard use case, aka writing ``<p>MyComment</p>``. We could even extract it into a variable and test that test_partner.comment == body_html .

Reply:
Done changes.